.gitignore

```diff
@@ -2,3 +2,5 @@
 submariner-engine
 .dapper
 bin
+.project
```

Comment:
Perhaps this should go in a separate patch (dedicated to ignoring Eclipse files). You can also configure git globally to ignore files specific to your IDE ;-).